src/comps/Header.tsx

```diff
@@ -17,6 +17,9 @@ import {
 import ConnectWallet from "./ConnectWallet";
 import { useWallet } from "@/util/WalletContext";
 import { NotificationStatusType } from "./Notifications";
+import { GetTestnetBTC } from "./get-testnet-btc";
+
+const isTestnet = process.env.NEXT_PUBLIC_WALLET_NETWORK !== "mainnet";
```

Comment:
Since this is used to enable the regtest faucet button and that only works for sBTC testnet, I'd check that the variable is set to `sbtcTestnet` specifically.

Reply:
fixed